cypress/integration/browse/new_user_and_camera.ts

```diff
@@ -9,28 +9,27 @@ context('New User', () => {
     cy.createGroup(group);
 
     // create a camera in the group
-    cy.createCamera(camera, group);
+    cy.apiCreateCamera(camera, group);
 
     // check that user can see camera
-    cy.visit('devices');
-    cy.contains(camera);
+    cy.checkDeviceInGroup(camera,group)
   });
 
 
   it('A camera event can be triggered', () => {
     const eventType = "throttle";
```

Comment:
This was done with the real camera code.  So probably doesn't make much sense as other than an api test unless we are using the real camera code.